VERSION

```diff
@@ -1 +1 @@
-v0.27.0-dev
\ No newline at end of file
+0.0.1
```

Comment:
Please, revert this change. 
You have already released a lot of version <=v0.26.0. See https://github.com/gardener/logging/releases/